safe/impact_functions/generic/classified_polygon_people/impact_function.py

```diff
@@ -226,10 +258,12 @@ def evaluate_impact(
         :rtype: []
         """
         # Taking area necessary attributes
-        area_id_attribute = self.exposure.keyword('field')
-        type_attr = self.exposure.keyword('area_population_field')
+        area_id_attribute = self.exposure.keyword('area_id_field')
```

Comment:
Can you clarify if you are expecting the user the specify an Area Name or and Area ID? If the former then for consistency, calling the variables area_name_field rather than area_id_field might be good.